gradle.properties

```diff
@@ -1,5 +1,5 @@
 springVersion=4.3.10.RELEASE
-springBootVersion=1.5.10.RELEASE
+springBootVersion=1.5.12.RELEASE
```

Comment:
Is this upgrade needed for docker compose to work?

Reply:
It is needed because sprintboot 1.5.10 doesn't exist in MVN repo anymore.

Reply:
It's possible I'm looking in the wrong place, but I see these:
https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-gradle-plugin/1.5.10.RELEASE
https://mvnrepository.com/artifact/org.springframework.boot/spring-boot/1.5.10.RELEASE

Also, the other Spinnaker services all seem to still be using `1.5.10` (or earlier) and I'm uncomfortable moving just this one to a later version with a compelling reason (I checked a handful of the others).

Lastly, it builds just fine for me, for a bunch of others, and in our nightly builds, so it seems like there isn't an issue resolving the currently-referenced version.